costs/model-costs.json

```diff
@@ -1,28 +1,14 @@
 [
+
+    
     {
-        "model": "gemini-1.5-pro",
-        "model_family": "gemini",
-        "request_context_window": 2000000,
-        "model_type": "text", 
-        "max_tokens_per_minute": 4000000, 
-        "max_requests_per_minute": 1000, 
-        "max_input_tokens": 2097152, 
-        "max_output_tokens": 8192,
-        "input_cost_10k": 0.025, 
-        "output_cost_10k": 0.10,
-        "developer": "Google DeepMind", 
-        "provider": "Google Cloud Platform", 
-        "pricing_page": "https://ai.google.dev/pricing#1_5pro", 
-        "pricing_plan": "Pay-as-you-go", 
-        "updated": "2024-10-25"
-    },
-    {
+        "s/n" : 1,
         "model": "gemini-1.5-pro",
```

Comment:
Gemini usually prefix their models with "models/", get the correct value from the models list page and reference that page here.